source/trainer.py

```diff
@@ -0,0 +1,147 @@
+# RUN 3 lines below in a seperate cell in Google Colab
+# !pip install transformers tokenizers wandb huggingface_hub
+# from huggingface_hub import notebook_login
+# notebook_login()
+
```

Comment:
If you wanna run this on colab, shouldn't this file be a .ipynb ?

Reply:
We will import the scripts from the cloned repo directly in collab

Reply:
ok makes sense